libs/core/src/lib/table/directives/table-body.directive.ts

```diff
@@ -7,4 +7,14 @@ export class TableBodyDirective {
     /** @hidden */
     @HostBinding('class.fd-table__body')
     fdTableBodyClass: boolean = true;
+
+    /** Whether or not to show the table body's horizontal borders */
+    @HostBinding('class.fd-table__body--no-horizontal-borders')
+    @Input()
+    noBorderX: boolean = false;
```

Comment:
I agree, but in this case I think the change needs to start from styles because the class that's being added is a negative.  So let's address this separately after that if we really think it's needed